app/Http/Controllers/CompetitionsController.php

```diff
@@ -7,7 +7,7 @@
 
 class CompetitionsController extends Controller
 {
-    private $validationRules = [
+    public static $validationRules = [
```

Comment:
@deadlybutter @weerd I think this would be a good scenario for a shared [Form Request](https://laravel.com/docs/5.2/validation#form-request-validation) that you could use from both controllers.